### DISCUSSION TITLE
TreePicker value enable for children

### DISCUSSION THREAD

**Discussion by mayanksolan** [rsuite/rsuite#3071] [Q&A]
For the treepicker, if the initial value on load is a parent value, it shows, but if the selected value is a child, it does not.

**Comment by SevenOutman** [comment]
@simonguo This is another advocation that Picker should display the "having value" state event if `value` is not present in `data` prop. What do you think?

**Comment by simonguo** [comment]
If there is no option corresponding to `value` in `data`, you can use `renderValue` to customize the display.
